### PR TITLE
Eliminated @GrpcService; bump to 0.0.7-SNAPSHOT

### PR DESCRIPTION
**Pull request by ronsigal**
I borrowed Stuart's treatment here, which is to search for all implementations of BindableService, and I added a test that an instantiation of BindableService is not overridden by another class.

### CODE REVIEW

**Comment by ronsigal** [comment]
Note, btw, that 

1. the api submodule is gone, and 
2. GreeterServiceImpl and ChatServiceImpl no longer have @GrpcService annotation.

**Comment by ronsigal** [comment]
Thanks, @hpehl, this was a fun one!

I need to look at the examples  ... I think @jamezp' latest update affected them. And then update the documentation.

**Comment by ronsigal** [comment]
I've made some updates:

1. updated WildFly location in chat and helloworld examples
2. added SSL files so that chat works
3. updated README.md and docs/guide/index.adoc

**Comment by ronsigal** [comment]
At this point, I think we're ready for a release. Should it be an Alpha or a Beta release?

File: pom.xml [hunk 1]
```diff
@@ -78,7 +78,7 @@
             <id>ossrh</id>
             <url>${repo.sonatype.url}/service/local/staging/deploy/maven2/</url>
         </repository>
-    </distributionManagement>
+    </distributionManagement-->
```
[the remark this answers is not in the document]
Comment (hpehl) [reply pom.xml:81]: @jamezp Sorry, I missed that. If you could fix that, that would be great. Thanks!

[the remark this answers is not in the document]
Comment (jamezp) [reply pom.xml:81]: @hpehl Done https://github.com/wildfly-extras/wildfly-grpc-feature-pack/pull/88

**Pull request merged**